File: 240227/Next Level/next-level.js
const fs = require("fs");
let [_id, _lv] = fs.readFileSync(0).toString().trim().split(" ");

class User {
    constructor(id = 'codetree', lv = 10) {
        this.id = id
        this.lv = lv
    }
}

const user = new User()
console.log(`user ${user.id} lv ${user.lv}`)

user.id = _id
user.lv = _lv
console.log(`user ${user.id} lv ${user.lv}`)

File: 240227/Next Level/next-level.py
class User:
    def __init__(selft, id="codetree", lv=10):
        selft.id = id
        selft.lv = lv

user1 = User()

user2_id, user2_lv = tuple(input().split())
user2 = User(user2_id, int(user2_lv))

print(f"user {user1.id} lv {user1.lv}")
print(f"user {user2.id} lv {user2.lv}")